gm4_audere_shamir/data/load/tags/functions/gm4_metallurgy.json

```diff
@@ -0,0 +1,3 @@
+{
+    "values": []
+}
```

Comment:
cc @misode 
This empty function tag is fine as metallurgy won't be bundled with this module by the website, correct?

Reply:
Yes that's correct. Currently all modules have empty tags for their dependencies